src/suites/cts/validation/vertex_input.spec.ts

```diff
@@ -564,9 +652,15 @@ g.test('check attribute offset out of bounds', async t => {
     const descriptor = t.getDescriptor(vertexInput, VERTEX_SHADER_CODE_WITH_NO_INPUT);
     t.device.createRenderPipeline(descriptor);
   }
+  {
+    // Control case, setting attribute offset to 4
+    vertexInput.vertexBuffers![0].attributeSet![0].offset = 4;
+    const descriptor = t.getDescriptor(vertexInput, VERTEX_SHADER_CODE_WITH_NO_INPUT);
+    t.device.createRenderPipeline(descriptor);
+  }
   {
     // Test attribute offset out of bounds
-    vertexInput.vertexBuffers![0].attributeSet![0].offset = MAX_VERTEX_BUFFER_END - 1;
+    vertexInput.vertexBuffers![0].attributeSet![0].offset = MAX_VERTEX_BUFFER_END - 3;
```

Comment:
Hmm, if offsets must be multiples of 4 as you said in the other comment, then this is triggering two errors (OOB, and multiple of 4). To avoid that you could do float2 and MAX_VERTEX_BUFFER_END-4.

Reply:
Done.